src/appmixer/linear/core/DeleteComment/component.json

```
@@ -0,0 +1,62 @@
+{
+    "name": "appmixer.linear.core.DeleteComment",
+    "author": "Appmixer <info@appmixer.com>",
+    "description": "Deletes an existing comment by its ID.",
+    "version": "1.0.0",
+    "private": false,
+    "auth": {
+        "service": "appmixer:linear"
+    },
+    "quota": {
+        "manager": "appmixer:linear",
+        "resources": "requests"
+    },
+    "inPorts": [
+        {
+            "name": "in",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "commentId": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "commentId"
+                ]
+            },
+            "inspector": {
+                "inputs": {
+                    "commentId": {
+                        "type": "text",
+                        "tooltip": "The ID of the comment to delete",
+                        "label": "Comment Id",
+                        "index": 0
+                    }
+                }
+            }
+        }
+    ],
+    "outPorts": [
+        {
+            "name": "out",
+            "options": [
+                {
+                    "label": "Success",
+                    "value": "success",
+                    "schema": {
+                        "type": "boolean"
+                    }
+                },
+                {
+                    "label": "Deleted Comment Id",
+                    "value": "deletedCommentId",
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ]
+        }
+    ],
```

Update and Delete components should return empty object.
```suggestion
    "outPorts": ["out"],
```